### PR TITLE
Command overhaul

### PR DESCRIPTION
**Pull request by Z1Meme**

### CODE REVIEW

**Comment by QuentinGruber** [comment]
We should put a deprecation message when you try to use /hax kit for example => "Hi this is deprecated use /kit" for some time. Users would be lost. @Z1Meme

**Comment by Z1Meme** [comment]
> We should put a deprecation message when you try to use /hax kit for example => "Hi this is deprecated use /kit" for some time. Users would be lost. @Z1Meme

Done 👍

**Comment by Z1Meme** [comment]
@QuentinGruber This is ready now

File: src/utils/utils.ts [hunk 1]
```diff
@@ -470,3 +470,16 @@ export function calculateDamageDistFallOff(
   //return damage / (distance * range);
   return damage * Math.pow(range, distance / 10);
 }
+
+export function flhash(str: string) {
```
Comment (QuentinGruber) [inline src/utils/utils.ts:474]: I will implement this hash func in h1emu-core

**Review by QuentinGruber — APPROVED** [review]

**Pull request merged**